src/tools/kata-ctl/src/utils.rs

```diff
@@ -5,7 +5,10 @@
 
 #![allow(dead_code)]
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, Context, Result};
+use std::fs;
+use std::io::Write;
+use tempfile::tempdir;
```

Comment:
To overcome the static errors and according to the doc, you should remove this line: `use tempfile::tempdir;`